### PR TITLE
New package: offpunk-1.5

### PR DESCRIPTION
**Pull request by Felecarpp**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/offpunk/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+maintainer="Felecarp <felecarp@tuta.io>"
+license="BSD-2-Clause"
+homepage="https://notabug.org/ploum/offpunk"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=2d775af28397c6987d5c84b2f529b69f3c332dc46f2f4992620d86d80377392b
+
+offpunk-web_package() {
+	short_desc+ " - web browsing enabled"
+	build_style=meta
+	depends="offpunk less file xdg-utils python3-cryptography python3-requests python3-readability-lxml python3-feedparser chafa timg"
```
Comment (classabbyamp) [inline srcpkgs/offpunk/template:19]: no need to duplicate the deps between subpkgs

File: srcpkgs/offpunk/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	depends="offpunk python3-requests python3-readability-lxml python3-feedparser chafa timg"
+}
+
+offpunk-full_package() {
+	short_desc+=" - all optional dependencies included"
+	build_style=meta
+	depends="offpunk-web xsel python3-setproctitle"
+}
+
+do_install() {
```
Comment (classabbyamp) [inline srcpkgs/offpunk/template:28]: this should be before `*_package()` functions

**Comment by ahesford** [comment]
These subpackages should be removed. These are optional dependencies, which XBPS doesn't support, and abusing meta-packages to fake optional deps isn't really appropriate.

**Comment by ahesford** [comment]
On further inspection, this isn't really suitable for packaging because it doesn't satisfy the quality requirements. It's a single python file you can just fetch, or install via pip, and run as desired.

**Comment by Felecarpp** [comment]
The install instructions I use
```
xbps-install python3-pipx less file xdg-utils chafa xsel
pipx install https://notabug.org/ploum/offpunk/archive/master.tar.gz
pipx runpip offpunk install -r https://notabug.org/ploum/offpunk/raw/master/requirements.txt
```

**Pull request closed without merging**